snap/gui/liri-text.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Liri Text
+Comment=Edit text files
+Exec=liri-text %U
+Terminal=false
+Type=Application
+Categories=Qt;Utility;TextEditor;
+Keywords=Text;Editor;
+Icon=${SNAP}/share/icons/hicolor/256x256/apps/io.liri.Text.png
```

Comment:
is it possible to reuse the existing .desktop file provided by text?

Reply:
> it possible to reuse the existing .desktop file provided by text?

Hey, I think the only problem is the `Icon` field.

Reply:
@plfiorini we could probably use the existing file and `sed` change the `Icon=...` line. That might be better.

Reply:
It would be better if you think it's possible, because the desktop entry provided by Text will eventually include the translations and we don't want to maintain two desktop entries.

Reply:
@plfiorini yep, will try.

Reply:
All right, should be done now :)